exec/java-exec/src/main/java/org/apache/drill/exec/store/pcap/decoder/Packet.java

```diff
@@ -324,7 +333,12 @@ public int getDst_port() {
     byte[] data = null;
     if (packetLength >= payloadDataStart) {
       data = new byte[packetLength - payloadDataStart];
-      System.arraycopy(raw, ipOffset + payloadDataStart, data, 0, data.length);
+      try {
+        System.arraycopy(raw, ipOffset + payloadDataStart, data, 0, data.length);
+      } catch (Exception e) {
+        isCorrupt = true;
+        logger.info("Error while parsing PCAP data: ", e.getMessage());
```

Comment:
I think log info will produce error for each corrupt row and log file can grow enormously. I guess this should be debug, you can also include trace for the full exception:

```
String message = "Error while parsing PCAP data: ";
logger.debug(message, e.getMessage());
logger.trace(message, e);
```

Reply:
Thanks @arina-ielchiieva.  Fixed.